### PR TITLE
add a debug version of libg package

### PR DESCRIPTION
**Pull request by pinzart90**
### Purpose

(FILL ME IN) This section describes why this PR is here. Usually it would include a reference to the tracking task that it is part or all of the solution for.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. Use N/A to indicate that the changes in this pull request do not apply to Release Notes. **Mandatory section**

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -33,7 +33,8 @@
     <PackageReference Include="Autodesk.IDSDK" Version="1.2.4" />
     <PackageReference Include="Greg" Version="3.0.2.6533" />
     <PackageReference Include="DynamoVisualProgramming.LibG_230_0_0" Version="3.0.0.6705" GeneratePathProperty="true" />
```
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:35]: If this is meant for debugging libg on Linux, we may also need a debug package for 230, at least until we update libg to 231 on DaaS.

Comment (pinzart90) [reply src/DynamoCore/DynamoCore.csproj:35]: Ugh, if possible ai would prefer to just switch DAAS to 231. If no breaking changes are expected

**Comment by mjkkirschner** [comment]
hmm I guess that debug builds of Dynamo were always slower than release builds, but now they might be even slower, correct?

**Comment by pinzart90** [comment]
> hmm I guess that debug builds of Dynamo were always slower than release builds, but now they might be even slower, correct?

Slower at build time and runtime. I don't know by how much

**Review by mjkkirschner — APPROVED** [review]
looks good to me but perhaps worth letting the teams know about.

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**